### PR TITLE
Temporary password sending via email when creating user

### PR DESCRIPTION
**Pull request by KasunJayasanka**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "@aws-sdk/client-s3": "^3.496.0",
     "@aws-sdk/lib-dynamodb": "^3.478.0",
     "@aws-sdk/lib-storage": "^3.496.0",
-    "bcrypt": "^5.1.1",
+    "bcryptjs": "^2.4.3",
```
Comment (madhurajayashanka) [inline package.json]: ?

**Review by KasunJayasanka — COMMENTED** [review]
corrected

**Comment by madhurajayashanka** [comment]
not yet. refine all

**Comment by KasunJayasanka** [comment]
> not yet. refine all

Changed again

**Pull request closed without merging**